lib/stripe/connect/account.ex

```diff
@@ -110,74 +143,76 @@ defmodule Stripe.Account do
           fields_needed: [String.t()]
         }
 
-  @type business_profile :: %{
-          mcc: String.t() | nil,
-          name: String.t() | nil,
-          product_description: String.t() | nil,
-          support_address: %{
-            city: String.t() | nil,
-            country: String.t() | nil,
-            line1: String.t() | nil,
-            line2: String.t() | nil,
-            postal_code: String.t() | nil,
-            state: String.t() | nil
-          },
-          support_email: String.t() | nil,
-          support_phone: String.t() | nil,
-          support_url: String.t() | nil,
-          url: String.t() | nil
-        }
-
-  @type capabilities :: %{
-          card_payments: String.t() | nil,
-          legacy_payments: String.t() | nil,
-          platform_payments: String.t() | nil
-        }
-
   @type t :: %__MODULE__{
-          id: Stripe.id(),
-          object: String.t(),
+          id: Stripe.id() | nil,
+          object: String.t() | nil,
+          business_logo: String.t() | nil,
```

Comment:
looks like some of these `business_` attributes don't exist anymore. Let me know if you would like to put in a fix!

https://stripe.com/docs/api/accounts/object